libs/island-ui/contentful/src/lib/RichTextRC/defaultRenderComponents.tsx

```diff
@@ -30,10 +31,13 @@ const renderConnectedComponent = (slice) => {
     case 'Skilavottord/CompanyListConnected':
       if (typeof data === 'object') {
         const { graphqlLink } = data
-
         return <CompanyListConnected graphqlLink={graphqlLink} />
       }
       break
+    case 'Fiskistofa/ShipSearch':
+      return <ShipSearch {...data} />
+    case 'Fiskistofa/ShipSearchSidebarInput':
+      return <SidebarShipSearchInput {...data} />
```

Comment:
Feel like all of this should be in `libs/web/richtext` or something, rather than inside the island-ui folder.

Know this isn't your fault but maybe you'll feel an itch to refactor this someday 😆